server/models/post.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const partitionPromise = require('@quarterto/partition-promise');
+const transform = require('../lib/transform');
+const articleModel = require('./article');
+const database = require('../lib/database');
+const getCanonical = require('./canonical');
+const fbApi = require('../lib/fbApi');
+const mode = require('../lib/mode');
+
+exports.get = async function get() {
+	const since = await database.getLastABCheck();
+	await database.setLastABCheck(Date.now()); // set this as soon as possible because this might take a while
+	if(since) {
+		return Promise.all(
+			(await fbApi.posts({since}))
+				.map(async function mapPosts(url) {
+					const canonical = await getCanonical(url);
+					return articleModel.get(canonical);
```

Comment:
Lots of work happens here, including hitting the content API to get the full article contents. I think we only need the canonical URL at this stage? Better to hydrate the article later.